operators/multiclusterobservability/manifests/base/grafana/dash-acm-virtual-machines-by-time-in-status.yaml

```
@@ -0,0 +1,425 @@
+              "format": "table",
+              "hide": false,
+              "instant": true,
+              "interval": "",
+              "legendFormat": "",
+              "refId": "B"
+            }
+          ],
+          "title": "Virtual Machines List by Time In Status",
```

Similar suggestions, this should be sentence-case
```suggestion
          "title": "Virtual machines list by time In status",
```